src/net_processing.h

```diff
@@ -32,6 +32,9 @@ struct CNodeStateStats {
     uint64_t m_addr_processed = 0;
     uint64_t m_addr_rate_limited = 0;
     bool m_addr_relay_enabled{false};
+    // Note: If you add fields to this struct, you should also consider updating
+    // the getpeerinfo RPC (in rpc/net.cpp), and the IPC serialization code (in
+    // ipc/capnp/node.cpp and ipc/capnp/node.capnp).
```

Comment:
It's unclear to me what need to be updated in `ipc/capnp/node.cpp` in case of new members added to `CNodeStateStats` ? Also for better clarity the object wrapper could be quoted `NodeStateStats`. Here and for `CNodeStats` and `proxyType` comments.